### ISSUE TITLE
Convert VMEC wout from text to nc

### ISSUE THREAD

**Issue by rmchurch** [PrincetonUniversity/STELLOPT#225]
I'm trying to go from a VMEC output file that is text based to NetCDF. I thought it would be as simple as:
```
call read_wout_file('wout_myfile',ierr)
call write_wout_file('wout_myfile.nc',ierr)
```
But I get a segfault at line 2001 in `read_wout_mod.F90`. I look through and don't see where `qfact` would be defined in the `read_wout_text`. Is there a step I am missing, or is this not possible? The wout file is VMEC v6.90
https://github.com/PrincetonUniversity/STELLOPT/blob/512375ce3582efa57910c336c83409577d760d6b/LIBSTELL/Sources/Modules/read_wout_mod.f90#L2001

**Comment by lazersos** [comment]
Just created a pull request to fix this (#226).  @rmchurch Let me know if this fixes your issue and I'll merge the pull request.

**Comment by rmchurch** [comment]
Tried out PR #226 and it fixes the issue, thanks.
